parm/config/config.anal

```diff
@@ -54,73 +54,70 @@ export AMSR2BF=${AMSR2BF:-/dev/null}
 
 # Set default values for info files and observation error
 #  NOTE:  Remember to set PRVT in config.prep as OBERROR is set below
-export CONVINFO=$FIXgsi/global_convinfo.txt
-export OZINFO=$FIXgsi/global_ozinfo.txt
-export SATINFO=$FIXgsi/global_satinfo.txt
-export OBERROR=$FIXgsi/prepobs_errtable.global
+export CONVINFO=${FIXgsi}/global_convinfo.txt
+export OZINFO=${FIXgsi}/global_ozinfo.txt
+export SATINFO=${FIXgsi}/global_satinfo.txt
+export OBERROR=${FIXgsi}/prepobs_errtable.global
 
 
 # Use experimental dumps in EMC GFS v16 parallels
-if [[ $RUN_ENVIR == "emc" ]]; then
+if [[ ${RUN_ENVIR} == "emc" ]]; then
 	export ABIBF="/dev/null"
-	if [[ "$CDATE" -ge "2019022800" ]] ; then
-		export ABIBF="$DMPDIR/${CDUMP}x.${PDY}/${cyc}/atmos/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
-		if [[ "$CDATE" -ge "2019111000" && "$CDATE" -le "2020052612" ]]; then
-			export ABIBF="$DMPDIR/${CDUMP}y.${PDY}/${cyc}/atmos/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
-		fi
+	if [[ "${CDATE}" -ge "2019022800" ]] ; then
+		export ABIBF="${COMIN_OBS}/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
 	fi
 
 	export AHIBF="/dev/null"
-	if [[ "$CDATE" -ge "2019042300" ]]; then
-		export AHIBF="$DMPDIR/${CDUMP}x.${PDY}/${cyc}/atmos/${CDUMP}.t${cyc}z.ahicsr.tm00.bufr_d"
+	if [[ "${CDATE}" -ge "2019042300" ]]; then
+		export AHIBF="${COMIN_OBS}/${CDUMP}.t${cyc}z.ahicsr.tm00.bufr_d"
```

Comment:
The changes are consistent w/ the PR, but I am not sure this is entirely correct.
This is going to be true, if script that gets the obs copies them into the `COMIN_OBS`. But if it doesn't, we have to go to the `DMPDIR`.
May be @RussTreadon-NOAA might know the details of these conditionals